plotting/ft_plot_sens.m

```diff
@@ -184,7 +184,7 @@
     case 'ctf151'
       senssize = 15; % FIXME this is only an estimate
```

Comment:
@ualsbombe can you change this to 1.999 for the CTF151

Reply:
Just to be clear - that would be 19.99 in the code, right, because everything is in millimetres, I assume?

Reply:
yes. And I propose you make it 20, because that is probably according to design and the difference might be a rounding off error. Round numbers are easier to remember.

Reply:
I've changed it to 20 now and pushed to my fork.